File: Backend/LeaveManagement/LeaveManagement/Application/DTOs/LeaveBalanceResponseDTO.cs
namespace LeaveManagement.Application.DTOs;
public class LeaveBalanceResponseDTO
{
    public int EmployeeId { get; set; }
    public string? FirstName { get; set; }
    public string? LastName { get; set; }
    public DateOnly Year { get; set; }
    public int Casual { get; set; }
    public int Sick { get; set; }
    public int Vacation { get; set; }
    public int Medical { get; set; }
}

File: Frontend/LeaveManagement/src/app/features/LeaveBalance/Dtos/LeaveBalanceResponse.dto.ts
export interface LeaveBalanceResponseDto {
    employeeId: number;
    firstName?: string; // string? in C# maps to optional string in TypeScript
    lastName?: string;  // string? in C# maps to optional string in TypeScript
    year: string;       // DateOnly (representing just the year) maps to string
    casual: number;
    sick: number;
    vacation: number;
    medical: number;
  }

  